packages/smooth_app/lib/cards/product_cards/knowledge_panels/knowledge_panels_builder.dart

```diff
@@ -15,6 +15,7 @@ class KnowledgePanelsBuilder {
     KnowledgePanels knowledgePanels, {
```

Comment:
unrelated comments on the file, sorry I missed the last review on it.

 I just saw `dummyAddButton` why is it dummy? raises a few questions.

Other comments: We should be able to put the add/edit buttons without special coding it (it will come as part of the api response if a button should be added - I'll have a check if it's done already on the server).,

Generally, the idea with KnowledgePanels was never to special case anything.

Reply:
It used to be dummy because it was just a button with no action, but used in different places. As a first step.
Then I started to code for one occurrence of the button, and someone else developed in parallel on another occurrence, therefore I could not quickly rename the button without impacting the other development.
But you're right, the name should be changed.